nwbwidgets/misc.py

```diff
@@ -106,7 +106,11 @@ def show_session_raster(
         progress_bar=progress_bar,
     )
     ax.set_ylabel("unit #")
-    ax.set_yticklabels(units.id.data[:])
+    if len(data) <= 30:
+        unit_id_display = np.array(units.id.data[:])[[x for x in this_iter]]
+        ax.set_yticklabels(unit_id_display)
+    else:
+        ax.axes.yaxis.set_visible(False)
```

Comment:
There's no good way to view the unit ids if they aren't consecutive and more than 30 in number, so just removing the labels altogether is best imo. Could add a test to see if they are at least consecutive if you want